features-json/resizeobserver.json

```
@@ -231,7 +231,7 @@
       "11.1":"n",
       "12":"n",
       "12.1":"n",
-      "13":"y",
+      "13":"n",
```

```suggestion
      "13":"n d",
```
If the support is behind a flag, this should include `d`